doc/ref_cert/lfn/chapters/chapter10.md

```diff
@@ -5,16 +5,23 @@
 
 ## Table of Contents
 * [10.1 Introduction](#10.1)
-* [10.2 Heading](#10.2)
-* [10.3 Heading](#10.3)
+* [10.2 RM/RA-1 Requirements](#10.2)
+* [10.3 TC Mapping to Requirements](#10.3)
 
 <a name="10.1"></a>
 ## 10.1 Introduction
 
+Define, and describe the purpose of this chapter to be:
+
+- Define RM/RA-1 Openstack requirements
+- Map Framework to Requirements
 
 <a name="10.2"></a>
-## 10.2 Heading
+## 10.2 RM/RA-1 Requirements
 
+- List out reference to Test Cases Requirement (coming from Chapter 6) and map them to real project test scripts.
```

Comment:
@mf4716 Will VNF requirements (Ch06) be RA-1 dependent? I don't think it should.

Reply:
@deebhatia - RC-1, inclusive of NFVI and VNF (interconnection and interface validations w/ the NFVI) are based on RM-1 and RA-1 specs and requirements.  Therefore, while there can be some generic VNF requirements, for the purposes of our RC-1 effort, there needs to be a call-out, or identification, of requirements which map the RM-1/RA-1 specs.  Make sense?

Reply:
Shouldn't we categorise `generic VNF requirements` as `Required NFVI Capabilities` because we will document it in a way `NFVI must provide the capability to ....` making it an NFVI level requirement to validate?
To validate VNF compliance of any VNF, the used NFVI should be RC-1 NFVI compliant. This should be the prerequisite.

Reply:
You're absolutely right @deebhatia .

Reply:
@deebhatia , any other questions, or can we resolve this conversation?

Reply:
Nope. Resolved.